src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -129,6 +129,37 @@ public static int parseInsurancePlan(String insurancePlanId) throws ParseExcepti
         return trimmedInsurancePlanId;
     }
 
+    /**
+     * Parses {@code String claimId} into a valid claimId.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given claimId is invalid based on preset conventions.
+     */
+    public static String parseClaimId(String claimId) throws ParseException {
+        requireNonNull(claimId);
+        return claimId.trim();
+    }
+
+    /**
+     * Parses {@code String claimId} into a valid claimId.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the claim amount is not a valid positive integer
+     */
+    public static int parseClaimAmount(String claimAmount) throws ParseException {
+        requireNonNull(claimAmount);
+        int claimAmountInt;
+        try {
+            claimAmountInt = Integer.parseInt(claimAmount.trim());
+            if (claimAmountInt < 0) {
+                throw new ParseException("Unplayable, Claim Amount cannot be negative");
+            }
+        } catch (NumberFormatException e) {
+            throw new ParseException("Unplayable, claim Amount must be a positive integer");
```

Comment:
For lines 155 and 158, to standardise the capitalisation of claim amount? Maybe just keep claim amount as all small letters?

Reply:
yup I agree but I think its easier to see as Claim Amount in camelcase

Reply:
i updated all the messages later on, you can view in the latest PR.